transaction-history-manager/build.gradle.kts

```diff
@@ -4,17 +4,11 @@ plugins {
 
 dependencies {
     implementation(project(":libs"))
-    annotationProcessor("org.projectlombok:lombok")
-
     implementation("org.springframework.boot:spring-boot-starter-data-mongodb")
     implementation("org.springframework.boot:spring-boot-starter-validation")
-    implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("com.itextpdf:itext7-core:${Versions.I_TEXT_PDF}")
     implementation("org.springframework.cloud:spring-cloud-starter-openfeign:${Versions.OPENFEIGN}")
 
     testImplementation("org.wiremock:wiremock-standalone:${Versions.WIREMOCK}")
-
-    compileOnly("org.projectlombok:lombok")
-
     testImplementation("org.springframework.boot:spring-boot-starter-test")
```

Comment:
duplicate

Reply:
look in the plugin ` "com.qpay.spring.base"` and others plugin, then check your build.gradle